utils/cr_demo/run_demo.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+dmg_pool_list="dmg pool list"
+dmg_check_enable="dmg check enable"
+dmg_check_start="dmg check start"
+dmg_check_query="dmg check query"
+dmg_check_disable="dmg check disable"
+POOL_SIZE="4G"
+POOL_LABEL="tank"
+
+format_storage() {
+    hostlist=$1
+    dmg_cmd="dmg storage format --host-list=$hostlist"
+    echo "Command: $dmg_cmd"
+    eval "$dmg_cmd"
+}
+
+create_pool() {
+    pool_size=$1
+    pool_label=$2
+    dmg_cmd="dmg pool create --size=$pool_size --label=$pool_label"
+    echo "Command: $dmg_cmd"
+    eval "$dmg_cmd"
+}
+
+inject_fault_mgmt() {
+    pool_label=$1
+    fault_type=$2
+    dmg_cmd="dmg faults mgmt-svc pool $pool_label $fault_type"
+    echo "Command: $dmg_cmd"
+    eval "$dmg_cmd"
+}
+
+if [ $# == 0 ]
+then
+    echo "Please supply at least one hostname."
+    exit
+fi
+
+# Prepare hostnames separated by comma for format, etc.
+host_list=""
+for hostname in "$@"
+do
+    if [ -z "$host_list" ]
+    then
+        host_list=$hostname
+    else
+        host_list="$host_list,$hostname"
+    fi
+done
+
+# Pass 1: Pool is on ranks, but not in MS - Orphan pool
+echo "1. Format storage on $host_list. Hit enter..."
+read -r
+format_storage "$host_list"
+# Wait for the storage to be ready before creating a pool.
+sleep 5
+
+echo $'\n2. Create a 4GB pool. Hit enter...'
+read -r
+create_pool $POOL_SIZE $POOL_LABEL
+
+echo $'\n3. Remove PS entry on MS. Hit enter...'
```

Comment:
It may be helpful to describe what the test case to show. For example, for this one:

Generate orphan pool via removing the pool entry on MS with fault injection but keep related pool shards on targets.

Then there can be three different kinds of repair actions:
1. Recovery the orphan pool by default.
2. Destroy related pool shards to release space - CIA_TRUST_MS.
3. Ignore the inconsistency and go ahead - CIA_IGNORE.

For demo convenience, we can create three pools, and check them in parallel with CIA_INTERACT, then elect different repair actions for each of them.

Reply:
Thanks for the suggestions. I tested all four cases and looks like they're all working. (I used the latest of your PR: https://github.com/daos-stack/daos/pull/10798).

I have a quick question. I used the following command:
`dmg check start -p POOL_NONEXIST_ON_MS:CIA_INTERACT`
In this case, how is the user supposed to know `POOL_NONEXIST_ON_MS` and `CIA_INTERACT` (and the two other options above)? Are they in the command help?

Reply:
Sorry, there is still not document to describe the inconsistency classification and repair action. We need to add them in future. Currently, for your test, please refer to proto file: src/proto/chk/chk.proto
enum CheckInconsistClass
enum CheckInconsistAction